core-test/src/test/scala/org/bitcoins/core/psbt/PSBTSerializationTest.scala

```diff
@@ -0,0 +1,19 @@
+package org.bitcoins.core.psbt
+
+import org.bitcoins.testkit.util.BitcoinSUnitTest
+import scodec.bits._
+
+class PSBTSerializationTest extends BitcoinSUnitTest {
+
+  val validPsbts: List[ByteVector] = List(
```

Comment:
In general it is best to use `Vector` in scala for collections. It has the "least" amount of surprises in terms of pathological performance. See http://www.lihaoyi.com/post/BenchmarkingScalaCollections.html for more information on the various collections performance characteristics